OpenSim/Simulation/Model/Model.cpp

```diff
@@ -661,9 +661,15 @@ void Model::createMultibodyTree()
 
     auto bodies = getComponentList<Body>();
     for (auto& body : bodies) {
-        _multibodyTree.addBody( body.getAbsolutePathName(),
-                                body.getMass(), false,
-                                const_cast<Body*>(&body) );
+        try {
+            _multibodyTree.addBody( body.getAbsolutePathName(),
+                                    body.getMass(), false,
+                                    const_cast<Body*>(&body) );
+        } catch (const std::runtime_error& e) {
+            cout << "Duplicate Body names are not permitted. Please update "
+                 << "your model's .osim file accordingly." << std::endl;
```

Comment:
Could one get this message when building a model programmatically (rather than loading an .osim file)? Consider adding to the message something like "

Is it that names must not be duplicated, or that absolute path names may not be duplicated? The message makes sense for pre-4.0 models, but is not really correct for 4.0 models.

Is there a reason you chose to send a message to cout rather than throw a new exception with a more specific message?